libs/core-components/src/lib/Button/Button.test.js

```diff
@@ -117,15 +109,15 @@ describe('Button', () => {
         <Button data-testid="no button here">{TEST_TEXT}</Button>
       );
       const el = queryByTestId('no button here');
-      expect(el).toBeNull;
+      expect(el).toBeNull();
```

Comment:
Why are parentheses removed? I tried this in JavaScript on Portal, and it broke test cases ([e80f9de](https://github.com/TACC/Core-Portal/pull/640/commits/e80f9de1af9160ff8f73bfe906271957b0908d69)). But I don't find any TypeScript info about removing parentheses from functions.

Reply:
Parentheses weren't there and they were added. Not sure why they were missing in the first place.